src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import 'utils/global';
+@import 'utils/variables';
+@import 'blocks/product';
+@import 'blocks/stars';
```

Comment:
Verify that the files `utils/global.scss`, `utils/variables.scss`, `blocks/product.scss`, and `blocks/stars.scss` exist in the specified directories. Missing files will cause the SCSS compilation to fail.